src/pages/guides/getting-started/creating-your-first-plugin/index.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Creating your first plugin
+description: Create your first own Photoshop plugin using UXP.
+contributors:
+  - https://github.com/pklaschka
+---
+
+# Creating your first UXP-based Photoshop plugin
```

Comment:
I respectfully disagree:
1. since this is a getting started section, it isn't a format that we need to re-use. If in other (non-getting-started) tutorials, we need to re-use it, that works, but I wouldn't necessarily do that in this article
2. The article needs (both for readers and for SEO alike) a title that "screams" at you "Start here!". "Introduction" could be an introduction to anything (the fact that it's "under" the "Quick Start" is an artifact of the IMHO, non-optimal IA on the Guides pages, something that I intend on hopefully improving in the future)
3. "Introduction", to me, sounds like this is a mere introduction to a series of articles without any content in it. Something that this article isn't.